packages/preset-create-react-app/package.json

```diff
@@ -22,14 +22,14 @@
     "prepublishOnly": "yarn build"
   },
   "dependencies": {
-    "@storybook/node-logger": "^5.2.8",
-    "@types/webpack": "^4.41.0",
+    "@storybook/node-logger": "^5.3.0",
```

Comment:
do we need to upgrade this? this means that people stuck on 5.2 will install a second instance. not sure it matters but just wanted to double check